10up/component-library
# Fix: Losing focus inside tabs

### Description of the Change

- Updates demo to include focusable elements for future testing
- Adds `display: none;` to tabs that are hidden. As as https://w3c.github.io/aria-practices/examples/tabs/tabs-1/tabs.html

### Benefits

- Fixes https://github.com/10up/component-library/issues/18
- Fixes accessibility compliance

### Possible Drawbacks

None known.

### Verification Process

1. Open the demo page for tabs after compile
2. Navigate the page with tab and keyboard
3. Confirm that only the tabpanel and content inside the tabpanel are focusable
4. Confirm that focus continues in the natural page order after the panel content
5. Confirm that only parent tab from tab list is focused in reverse
6. Confirm that focus continues in reverse page order after tab list

### Checklist:






https://github.com/10up/component-library/issues/18
